src/zenml/steps/base_step.py

```diff
@@ -1070,7 +1070,7 @@ def _validate_inputs(
                 or key in client_lazy_loaders
             ):
                 continue
-            raise StepInterfaceError(f"Missing entrypoint input {key}.")
+            raise StepInterfaceError(f"Missing entrypoint input {key} in step `{self.name}`.")
```

Comment:
```suggestion
            raise StepInterfaceError(f"Missing entrypoint input '{key}' in step '{self.name}'.")
```

Reply:
This formats better.

Reply:
other StepInterface errors also use `{self.name}` - but I'm happy to change this - any particular reason you prefer ' ?

Reply:
@strickvl

Reply:
Try both out in the terminal and tell me which looks nicer :)

Reply:
@AlexejPenner